.pre-commit-config.yaml

```diff
@@ -7,7 +7,8 @@ repos:
       - id: end-of-file-fixer
         exclude: |
           (?x)(
-              tests/data/schema.gql
```

Comment:
docs ignore

Reply:
The syntax here is actually using `|`, so this should be:

```
test/data/schema.gql|
docs
```

But having said that, is there a reason to why docs can't have their end of file fixed?

Reply:
No reason really as I said earlier I'm not used to poetry or pre-commit so I was afraid that this is formmating the docs or somthing, because material for mkdocs is not pure .md.